WebRoot/templates/mail/Message.template

```diff
@@ -524,7 +555,10 @@
 					<$ var attachment = data.attachments[i]; $>
 
 					<span id='<$= attachment.spanId $>' class="AttachmentLoading attachmentBubble AttachmentSpan">
-						<span>
+						<span tabindex=0 onclick="<$= attachment.closeHandler $>" onkeypress="<$= attachment.closeHandler $>" class="AttachmentClose" role="button" aria-label="<$= ZmMsg.remove $>">
+					    <$= AjxImg.getImageHtml("BubbleDelete") $>
```

Comment:
Earlier thought about this line <span tabindex=0 onclick="<$= attachment.closeHandler $>" onkeypress= ..
Done for this.